vhdbuilder/packer/components.json

```diff
@@ -90,8 +90,6 @@
     {
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-npm:*",
       "amd64OnlyVersions": [
-        "v1.4.9",
```

Comment:
we should delete these and replace with 1.4.21 and 1.4.22, the current images for default and default in sov cloud respectively

Reply:
Thanks for sharing this. Does it need RP changes for the VHD image preload check as well? Who will be making those? I don't have much context about azure npm - making these changes mainly to reduce space for the VHD (so we can create a successful build for the weekly release)

Reply:
Discussed offline that we can go ahead with this PR and include default NPM images in another PR